GoogleForCreators/web-stories-wp
# Use register_block_type_from_metadata function

## Context



## Summary

The register_block_type_from_metadata function will be added in WordPress 5.5. The web stories plugin should use the register_block_type_from_metadata function and register on attributes in block.json.

## Relevant Technical Choices
I copied the block.json file into the assets/js directory so it would used on the built version of the plugin. 

## To-do



## User-facing changes



## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Create a new post
2. Insert web stories block. 
3. Block should work as before.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #3331
